src/main/java/emu/grasscutter/command/commands/QuestCommand.java

```diff
@@ -9,52 +9,59 @@
 
 import static emu.grasscutter.utils.Language.translate;
 
-@Command(label = "quest", usage = {"(add|finish) [<questId>]"}, permission = "player.quest", permissionTargeted = "player.quest.others")
+@Command(label = "quest", usage = {"(add|finish) [<questId>]", "enable"}, permission = "player.quest", permissionTargeted = "player.quest.others")
 public final class QuestCommand implements CommandHandler {
 
     @Override
     public void execute(Player sender, Player targetPlayer, List<String> args) {
-        if (args.size() != 2) {
+        if (args.size() == 0) {
             sendUsageMessage(sender);
             return;
         }
 
-        String cmd = args.get(0).toLowerCase();
-        int questId;
-
-        try {
-            questId = Integer.parseInt(args.get(1));
-        } catch (Exception e) {
-            CommandHandler.sendMessage(sender, translate(sender, "commands.quest.invalid_id"));
-            return;
-        }
-
+        String cmd = args.remove(0).toLowerCase();
         switch (cmd) {
-            case "add" -> {
-                GameQuest quest = targetPlayer.getQuestManager().addQuest(questId);
-
-                if (quest != null) {
-                    CommandHandler.sendMessage(sender, translate(sender, "commands.quest.added", questId));
+            case "enable":
+                targetPlayer.getQuestManager().enableQuests();
+                CommandHandler.sendMessage(sender, translate(sender, "commands.quest.enabled"));
+                break;
+            case "add":
+            case "finish":
+                int questId;
```

Comment:
maybe something like this before the switch case might be better, because the if else afterwards makes the code kinda ugly:
```suggestion
           if(args.size() == 1) {
                try {
                    questId = Integer.parseInt(args.get(0));
                }
                catch (Exception e) {
                    CommandHandler.sendMessage(sender, translate(sender, "commands.quest.invalid_id"));
                    return;
                }
           }
```

if later other parameters or options get added (like a mainQuestId via `m[0-9]+`, something similar to the spawnCommand handling could also be added before the final numbered argument handling.